moto/rds2/models.py

```diff
@@ -704,7 +704,8 @@ def create_database_replica(self, db_kwargs):
         if self.arn_regex.match(source_database_id):
             db_kwargs['region'] = self.region
 
-        replica = copy.deepcopy(primary)
+        # Shoudln't really copy here as the instance is duplicated. RDS replicas have different instances.
```

Comment:
Typo on the comment here. "Shoudln't" -> "Shouldn't"

Reply:
Haha :smile: